src/track/trackmetadata.cpp

```diff
@@ -66,4 +66,13 @@ QString TrackMetadata::reformatYear(QString year) {
     return year.simplified();
 }
 
+bool operator==(const TrackMetadata& lhs, const TrackMetadata& rhs) {
```

Comment:
It's still used, but only in a test. Yes, I already tried to remove it and since you have the same thought I will do it now! I forgot to update the plugins and broke the build, anyway ;)

Reply:
No, the eq-op is still used. When setting the metadata on a Track object all properties must be taken into account.